reference-architecture/aws-ansible/playbooks/openshift-install.yaml

```diff
@@ -7,6 +7,14 @@
   roles:
       # Group systems
       - instance-groups
+- hosts: bastion 
```

Comment:
Not directly related to this PR, but I would add some blank lines between the separate plays. It gets hard on the eyes to try to follow based on the indentation level alone.